packages/ui-extensions/src/surfaces/checkout/api/standard/standard.ts

```diff
@@ -1774,6 +1795,14 @@ export interface CustomerPrivacy {
    * Whether the visitor is in a region requiring data sale opt-outs.
    */
   saleOfDataRegion: boolean;
+  /**
+   * Details about the visitor's current location for use in evaluating if more granular consent controls should render.
+   *
+   * @example `{countryCode: 'CA', provinceCode: 'ON'}` for a visitor in Ontario, Canada; `{countryCode: 'US', provinceCode: undefined}` for a visitor in the United States if geolocation fails to detect the state; or `undefined` if neither country nor province is detected or geolocation fails.
+   *
+   * {% include /apps/checkout/privacy-icon.md %} Requires level 1 access to [protected customer data](/docs/apps/store/data-protection/protected-customer-data).
+   */
```

Comment:
It seems strange that these don't render in the docs. Are the examples mean for IDE type signatures only?

I see the protected customer data information render for country and province code, but no examples for them, either.